### DISCUSSION TITLE
话说，各位的烂尾楼贷款有人免除债务吗？

### DISCUSSION THREAD

**Discussion by caotianchao** [WeNeedHome/SummaryOfLoanSuspension#1141] [General]
[https://twitter.com/caijingxiang/status/1560473937980534784](url)

**Comment by Cegazhos** [comment]
银行不会免你的 政府也不会。
等开发商破产清算 或者打官司呗

**Comment by caotianchao** [comment]
人家就把非洲債務都免了，大手一揮。。。

**Reply by itsoursduty** [reply]
这下大撒币了

**Reply by caotianchao** [reply]
宁与外奴。。。

**Comment by volans19** [comment]
人家一带一路那些破烂儿国家几百亿几千亿的免，靠的是啥

**Reply by Cegazhos** [reply]
这些穷郭嘉 你还不能把他们逼急了。逼急了 他们政变 转头就不承认债务 甚至没收企业资产

**Comment by Cegazhos** [comment]
https://www.guancha.cn/economy/2022_08_20_654509.shtml
看样子是要拿财政兜底了
